source/ingress.go

```diff
@@ -139,12 +143,16 @@ func (sc *ingressSource) endpointsFromTemplate(ing *v1beta1.Ingress) ([]*endpoin
 		return endpoints, nil
 	}
 
+	ttl, err := getTTLFromAnnotations(ing.Annotations)
+	if err != nil {
+		log.Warn(err)
+	}
```

Comment:
Since you continue the loop here we should set `ttl` to something sane, e.g. `300`.

Reply:
i think it is okay to leave it as it is here, as this should be handled in providers (if they care about TTL field in endpoint)